frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1423,7 +1423,7 @@ function ReaderFooter:addToMainMenu(menu_items)
                             for _, letter in pairs(symbol_prefix.icons) do
                                 table.insert(sym_tbl, letter)
                             end
-                            return T(_("Icons (%1)"), table.concat(sym_tbl, " "))
+                            return T(C_("Status bar", "Icon (%1)"), table.concat(sym_tbl, " "))
```

Comment:
Is the change from Icons to Icon on purpose?

Reply:
Yep, isn't that what you meant? https://github.com/koreader/koreader/pull/9915#discussion_r1048990409